torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -1063,44 +1075,52 @@ def cached_autotune(
     ):
         configs_hash = hash_configs(configs)
 
+        local_cache = None
```

Comment:
Maybe these vars should be packed together in an object and the following code should be a separate function?